3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentRegistry.scala

```diff
@@ -184,6 +198,24 @@ case class EnrichmentRegistry(private val configs: EnrichmentMap) {
    */
   def getCampaignAttributionEnrichment: Option[CampaignAttributionEnrichment] = 
     getEnrichment[CampaignAttributionEnrichment]("campaign_attribution")
+  
+  /**
+   * Returns an Option boxing the UserAgentUtilsEnrichment
+   * config value if present, or None if not
+   *
+   * @return Option boxing the UserAgentUtilsEnrichment instance
+   */
+  def getUserAgentUtilsEnrichment: Option[UserAgentUtilsEnrichment.type] = 
+    getEnrichment[UserAgentUtilsEnrichment.type]("user_agent_utils")
```

Comment:
I think this should be "user_agent_utils_config" to match the name of the schema - otherwise how can the EnrichmentManager look up the enrichment in from the EnrichmentMap?